scripts/ts-function-release.sh

```diff
@@ -0,0 +1,56 @@
+#! /bin/bash
+#
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -euo pipefail
+
+scripts_dir="$(dirname "$0")"
+# git-tag-parser.sh has been shell-checked separately.
+# shellcheck source=/dev/null
+source ${scripts_dir}/git-tag-parser.sh
+
+DEV_TAG=dev
+versions=$(get_versions "${TAG}")
+GCR_REGISTRY=${GCR_REGISTRY:-gcr.io/kpt-functions}
+
+cd "${scripts_dir}/../functions/ts/${CURRENT_FUNCTION}"
+
+# This make it work for npm 6.*.*
+sed -i.bak "s|gcr.io/kpt-functions|${GCR_REGISTRY}|g" package.json
```

Comment:
Why are we editing package.json like this?

Can you just change registry in the functions from kpt-functions to new one?

Reply:
This is the reason: https://github.com/GoogleContainerTools/kpt/issues/1394

Reply:
Can you add a TODO here and follow up on that issue. There must be an alternative provided by NPM.  Hack here should be removed soon.